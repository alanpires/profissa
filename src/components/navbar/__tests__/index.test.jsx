import React from "react";
import Nav from "../nav";
import renderer from "react-test-renderer";

describe("Nav Test", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Nav open={true} showLogin={() => {}} setShowLogin={() => {}} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Nav open={false} showLogin={() => {}} setShowLogin={() => {}} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
