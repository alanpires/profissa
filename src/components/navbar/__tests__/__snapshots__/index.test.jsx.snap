// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Test renders correctly 1`] = `
<nav
  className="sc-dlnjPT jScYVP"
>
  <a
    href="/"
  >
    <div
      className="logo"
    >
      Profissa
    </div>
  </a>
  <div
    className="sc-gtssRu jnAMnz"
    open={true}
  >
    <div />
    <div />
    <div />
  </div>
  <ul
    className="sc-bdnylx faGLEd"
    open={true}
  >
    <testDiv
      className="div-usernavbar"
      onClick={[Function]}
    >
      <p>
        name
      </p>
      <img
        className="photo-navbar"
        src="userDefault.jpg"
      />
    </testDiv>
  </ul>
</nav>
`;

exports[`Nav Test renders correctly 2`] = `
<nav
  className="sc-dlnjPT jScYVP"
>
  <a
    href="/"
  >
    <div
      className="logo"
    >
      Profissa
    </div>
  </a>
  <div
    className="sc-gtssRu gWHJkT"
    open={false}
  >
    <div />
    <div />
    <div />
  </div>
  <ul
    className="sc-bdnylx bcEbrR"
    open={false}
  >
    <testDiv
      className="div-usernavbar"
      onClick={[Function]}
    >
      <p>
        name
      </p>
      <img
        className="photo-navbar"
        src="userDefault.jpg"
      />
    </testDiv>
  </ul>
</nav>
`;
